### ISSUE TITLE
Alterac Valley BG - Removed Quests

### ISSUE THREAD

**Issue by Swater96** [classicdb/database#370]
First AV Buglist: (https://github.com/classicdb/database/issues/48)

Patch 1.10.0 (www.wowwiki.com/Patch_1.10.0) (http://www.wowwiki.com/Category:Removed_in_patch_1.10.0)

Players in Alterac Valley will no longer drop Darkspear Troll Mojo, Dwarf Spines, Forsaken Hearts, Human Bone Chips, Orc Teeth, etc. and the quests have been removed for these items. Instead when a player is slain in Alterac Valley, any enemies that were in that player's threat list will gain 1 point of reputation with either the Frostwolf or Stormpike factions.

This would mean that the following quests should be removed, or unlinked from quest givers (to save for historical/nostalgic purposes) and the associated objects despawned. This is because some of the quest givers are actually still present and only the quests were removed.
Alliance:
- What the Hoof? (http://www.wowhead.com/quest=7424/)
- Wanted: ORCS! (http://www.wowhead.com/quest=7402/wanted-orcs)
- Wanted: MORE ORCS! (http://www.wowhead.com/quest=7428/wanted-more-orcs)
- The Quest-giving Sign: (http://www.wowhead.com/object=179437/wanted-orcs)
- The Archbishop's Mercy (http://www.wowhead.com/quest=7366)
- Staghelm's Requiem (http://www.wowhead.com/quest=7365)
- Defusing the Threat (http://www.wowhead.com/quest=7367)
- Gnomeregan Bounty (http://www.wowhead.com/quest=7364)

Horde:
- Wanted: DWARVES! (http://www.wowhead.com/quest=7401)
- Wanted: MORE DWARVES! (http://www.wowhead.com/quest=7427)
- The Quest-giving Sign: (http://www.wowhead.com/object=179438)
- Ally of the Tauren (http://www.wowhead.com/quest=7362)
- Defusing the Threat (http://www.wowhead.com/quest=7368)
- Favor Amongst the Darkspear (http://www.wowhead.com/quest=7361)
- The Human Condition (http://www.wowhead.com/?quest=7363)

NPC's (should be removed from "creature")
- Frostwolf Explosives Expert (http://www.wowhead.com/?npc=13597)
- Stormpike Explosives Expert (http://www.wowhead.com/?npc=13598)
- "Winterax ______" 12156,10984,12158,13956,13958,12157,10983,13957,11679

**Comment by Tobschinski** [comment]
No, I need to idle tunnel!

**Comment by Swater96** [comment]
Haha well as the quests sit now, most aren't even completable and are completely broken, so you might have had to....dare I say it...PvP in Alterac Valley either way. ;)

**Comment by Swater96** [comment]
```
-- Fixes for bug #370 (https://github.com/classicdb/database/issues/370) --
-- Written by Swater96 --
-- Deletes the Wanted! signs that gave now deleted quests --
DELETE from gameobject where id IN (179437,179438);
-- Deletes Winterax spawns from the AV graveyard, and Frostwolf/Stormpike Explosives Expert -- 
DELETE from creature where id IN (12156,10984,12158,13956,13958,12157,10983,13957,11679,13597,13598);
-- Removes quests that were removed in 1.10 from questrelation --
DELETE from creature_questrelation where quest IN (7424,7402,7428,7366,7365,7367,7364,7401,7427,7362,7368,7361,7363);
```

**Comment by cala** [comment]
Closed by https://github.com/classicdb/database/commit/06740d3d2a776ee65eb2e625896fdbc54109a220
